.gitignore

```diff
@@ -25,6 +25,7 @@ nbproject
 *.sublime-project
 *.sublime-workspace
 .idea/*
+.vscode/*
```

Comment:
I'm totally fine adding this here, but FYI, if you add this to [a global .gitignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html), you don't have to do it for each repo you work in.

Reply:
TIL, that's a good point! 👍